### ISSUE TITLE
don't allow mathematical transformations on string data

### ISSUE THREAD

**Issue by raprasad** [vjdorazio/TwoRavens#8]
- Currently users can select a variable such as `country` and attempt to apply a transformation such as `d^2`
- Provide better error messaging via the client or rook
- Currently provides message below:

---
 
<img width="492" alt="screenshot_8_15_17__4_58_pm" src="https://user-images.githubusercontent.com/1036060/29336244-27c7d4a0-81db-11e7-9038-9f84fd9a6895.png">
error message from rook:

**Comment by tercer** [comment]
Good point.  We should make better use of the type defintions of variables from the metadata, when we trust them, and check whether transformations are meaningful against those types.

Privacy Tools is doing something more sophisticated in this regard for it's own version of transformations.  It's implemented in Haskell.  Probably more (and clumsier) than we need, but perhaps as we integrate that code in for the Privacy case, we can learn what works and implement something better here.
